maze_bench_nim.nimble

```diff
@@ -14,5 +14,5 @@ requires "nim >= 1.6.0"
 requires "pixie >= 3.0.4"
 
 task benchmark, "Runs the benchmark":
-  exec "nim r -d:danger --opt:speed --gc:arc --passC:-flto --passL:-flto src/maze_bench_nim.nim"
+  exec "nim r -d:danger --opt:speed --gc:arc --passC:'-flto -march=native' --passL:-flto src/maze_bench_nim.nim"
```

Comment:
I don't really think it's a good idea to do `-march=native` - while it does allow the compiler to use CPU features available for the host CPU the binary becomes much less "portable" between machines, and unfair to other implementations that don't use the same feature.

Reply:
you're right, of course.  without -march=native, the generation phase goes up to 11ms for this version.